### PR TITLE
Inline delete rewrite

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#145

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite)

File: package.json [hunk 1]
```diff
@@ -46,37 +46,37 @@
...
+    "@skyux/inline-form": "4.1.0",
     "@skyux/layout": "4.2.3",
     "@skyux/list-builder-common": "4.0.1",
-    "@skyux/lists": "4.2.0",
+    "@skyux/lists": "4.3.0",
     "@skyux/lookup": "4.3.0",
     "@skyux/modals": "4.5.0",
     "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/popovers": "4.2.0",
+    "@skyux/popovers": "4.2.1",
```
Comment (Blackbaud-SteveBrush) [inline package.json:76]: - Do we need to update the popovers version in `peerDependencies`?
- Also, I would double-check the `@skyux/core` peer dependency version to make sure it is set to the version that the affix/overlay services were introduced.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-inline-delete-rewrite)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**